cli/tox.ini

```diff
@@ -22,16 +23,16 @@ commands =
 
 [testenv:py27-integration]
 commands =
-  py.test -x -vv {env:CI_FLAGS:} tests/integrations{posargs}
+  py.test -vv {env:CI_FLAGS:} tests/integrations{posargs}
 
 [testenv:py34-integration]
 commands =
-  py.test -x -vv {env:CI_FLAGS:} tests/integrations{posargs}
+  py.test -vv {env:CI_FLAGS:} tests/integrations{posargs}
 
 [testenv:py27-unit]
 commands =
-  py.test -x -vv {env:CI_FLAGS:} tests/unit{posargs}
+  py.test -vv {env:CI_FLAGS:} tests/unit{posargs}
 
 [testenv:py34-unit]
 commands =
-  py.test -x -vv {env:CI_FLAGS:} tests/unit{posargs}
+  py.test -vv {env:CI_FLAGS:} tests/unit{posargs}
```

Comment:
I think @tamarrow wanted this to fail early in TC. I can see the value to both approaches. Can we reach a compromise? This may complicated our build even more but maybe we can have TC fail early and developers can configure this however they wish. Thoughts?